integration-tests/scripts/install.sh

```diff
@@ -21,19 +21,45 @@ if [ -f "$HOME/rhtap-cli-ci-kubeconfig" ]; then
 fi
 
 echo "[INFO]Configuring deployment"
-acs_config="${acs_config:-new}" # new, hosted
-tpa_config="${tpa_config:-new}" # new, hosted
-registry_config="${registry_config:-quay}" # quay, quay.io, artifactory, nexus
-scm_config="${scm_config:-github}"  # github, gitlab, bitbucket
-pipeline_config="${pipeline_config:-tekton}" # tekton, jenkins
-auth_config="${auth_config:-github}" # github, gitlab
-
-echo "[INFO] acs_config=$acs_config"
-echo "[INFO] tpa_config=$tpa_config"
-echo "[INFO] registry_config=$registry_config"
-echo "[INFO] scm_config=$scm_config"
-echo "[INFO] pipeline_config=$pipeline_config"
-echo "[INFO] auth_config=$auth_config"
+if [[ -n "${acs_config:-}" ]]; then
```

Comment:
I think some folks are using this script locally for deploying rhads in their env.
I'm not sure we have somewhere documented what env vars and in what format this script excepts.
Could you please add it as it is not obviously clear from looking at this script I think...